_posts/patterns/2016-01-28-alerts.md

```diff
@@ -0,0 +1,275 @@
+                cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+            </div>
+        </div>
+
+        <div class="alert-action">
+            <button type="button" class="btn-neutral btn">Alert Action</button>
+        </div>
+    </div>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Success w/o Title</h3>
+<div class="example-set">
+    <div class="alert alert-success">
+        <div class="icon-display">
+            <span class="icon alert-icon icon-check" aria-hidden="true"></span>
+        </div>
+
+        <div class="alert-message">
+            <div class="alert-copy">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                laboris nisi ut aliquip ex ea commodo consequat.
+            </div>
+        </div>
+
+        <div class="alert-action">
+            <button type="button" class="btn-neutral btn">Alert Action</button>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Success w/o Title</h3>
+<div class="example-set">
+    <div class="alert alert-success">
+        <div class="icon-display">
+            <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+        </div>
+
+        <div class="alert-message">
+            <div class="alert-copy">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+            </div>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Warning w/ Title</h3>
+<div class="example-set">
+    <div class="alert alert-warning">
+        <div class="icon-display">
+            <span class="icon alert-icon icon-exclamation-triangle" aria-hidden="true"></span>
+        </div>
+
+        <div class="alert-message">
+            <div class="alert-title">
+                Alert Title!
+            </div>
+
+            <div class="alert-copy">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
+                in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
+                cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+            </div>
+        </div>
+
+        <div class="alert-action">
+            <button type="button" class="btn-neutral btn">Alert Action</button>
+        </div>
+    </div>
+</div>
+
+
+<h3 class="hd-6 example-set-hd">Warning w/o Title</h3>
+<div class="example-set">
+    <div class="alert alert-warning">
+        <div class="icon-display">
```

Comment:
See [my note about simplifying this above](https://github.com/edx/ux-pattern-library/pull/277/files#r51175181).